### ISSUE TITLE
Instructions on bare repo for pushing up from existing folder need to tell user to commit

### ISSUE THREAD

**Issue by pbx** [gitlabhq/gitlabhq#9587]
When you go to the home page of a project with an empty repo, the "Existing folder or Git repository" instructions are:
```
cd existing_folder
git init
git remote add origin gitlab@gitlab.example.com:repo.git
git push -u origin master
```
This probably works if you are pushing from an existing Git repo. However, if you execute the above verbatim in an existing folder that is not already a Git repo, the last command fails with: 
```
error: src refspec master does not match any.
error: failed to push some refs to gitlab@gitlab.example.com:repo.git
```
The missing piece is: if the user is creating a new git repo locally, they need to make a commit before trying to push.

**Comment by stanhu** [comment]
I believe this has been fixed via 773545d533ae4b9ab2b55d2e4942dc78cbe28f27.

**Comment by pbx** [comment]
Thanks.
